packages/elements/src/pagination/index.ts

```diff
@@ -315,15 +314,22 @@ export class Pagination extends BasicElement {
    * Getter for display text in the input
    * @returns input text
    */
-  protected get inputText (): string {
+  protected get inputValue (): string {
```

Comment:
Is it really input value or just display text or both ? This could be more readable and please update function description.

Reply:
Updated